doc/source/asyncio/guide/pattern.rst

```diff
@@ -0,0 +1,252 @@
+.. _asyncio-facade-pattern:
+
+AsyncIO Facade Pattern
+======================
+
+This section present a code pattern that you can use to migrate your
```

Comment:
Is the implied assumption is that you drop Eventlet altogether? If so you need to say so.

Also you switch from "migrate your application" to talking about libraries, those are very different use cases.